csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -150,9 +150,8 @@ function setUpMode(){
   }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/58442074-b9193000-813b-11e9-8984-9c78a0c57da2.png)
Separate the different grids somehow

Reply:
This version also still has console logs (?mode=edgedetection&picturepicker)